wallet/walletrpc.c

```diff
@@ -393,6 +393,102 @@ static const struct json_command newaddr_command = {
 };
 AUTODATA(json_command, &newaddr_command);
 
+static void json_listaddrs(struct command *cmd,
+						   const char *buffer, const jsmntok_t *params)
+{
+	struct json_result *response = new_json_result(cmd);
+	struct ext_key ext;
+	struct sha256 h;
+	struct ripemd160 h160;
+	struct pubkey pubkey;
+	jsmntok_t *bip32tok;
+	u8 *redeemscript;
+	bool ok;
+	char *out_p2sh;
+	char *out_p2wpkh;
+	const char *hrp;
+	u64 bip32_max_index;
+	const tal_t *tmpctx = tal_tmpctx(NULL);
+
+	if (!json_get_params(cmd, buffer, params,
+						 "?bip32_max_index", &bip32tok, NULL)) {
```

Comment:
Not preferred indentation.  Should be:

    if (!json_get_params(cmd, buffer, params,
                         "?bip32_max_index", &bip32tok,
                         NULL)) {

(the extreme indentation rather makes the newline useless)